WG LC preparation (step 5) (#102)
ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis: draft-ietf-ccamp-rfc9093-bis.md

> A YANG grouping to define the common capabilities attributes limit
range in case of operational mode and explicit mode.

modulation:

> > A base YANG identity the different modulation types, as defined in {{ITU-T_G.Sup39}}

Author: ```suggestion
> A base YANG identity for the modulation types, as defined in {{ITU-T_G.Sup39}}
```

> > A base YANG identity the different modulation types, as defined in {{ITU-T_G.Sup39}}

Author: ```suggestion
> A base YANG identity to define the different modulation types, as defined in {{ITU-T_G.Sup39}}
```

snr:

> A YANG data type used to represent an (Optical) Signal to Noise Ratio measured over 0.1 nm resolution bandwidth, as defined in {{ITU-T_G.977.1}}

psd:

> A YANG data type used to represent a Power Spectral Density (PSD), as defined in {{ITU-T_G.9700}}

penalty-value:

> > A YANG grouping to define the penalty value used of multiple penalty types, such as Chromatic Dispersion (CD), Polarization Mode Dispersion (PMD), as defined in {{ITU-T_G.666}} or Polarization Dependent Loss(PDL)

Author: ```suggestion
> A YANG grouping which defines the penalty value used of multiple penalty types, such as Chromatic Dispersion (CD), Polarization Mode Dispersion (PMD), as defined in {{ITU-T_G.666}} or Polarization Dependent Loss(PDL)
```

> > A YANG grouping to define the penalty value used of multiple penalty types, such as Chromatic Dispersion (CD), Polarization Mode Dispersion (PMD), as defined in {{ITU-T_G.666}} or Polarization Dependent Loss(PDL)

Author: ```suggestion
> A YANG grouping to define the penalty value for multiple penalty types, such as Chromatic Dispersion (CD), Polarization Mode Dispersion (PMD), as defined in {{ITU-T_G.666}} or Polarization Dependent Loss(PDL)
```
